DistributedTransaction/DistributedTransaction.tla

```diff
@@ -390,47 +381,20 @@ ServerCommit ==
 ServerCleanupStaleLock ==
   \E k \in KEY :
     \E l \in key_lock[k] :
-      \* the `resolving_pessimistic_lock` field is set to *TRUE*
-      CASE l.type \in {"lock_key"} ->
-        /\ SendReqs({[type |-> "check_txn_status_req",
+      /\ SendReqs({[type |-> "check_txn_status",
                     start_ts |-> l.ts,
-                    primary |-> l.primary, 
-                    rollback_if_not_exist |-> TRUE,
-                    resolving_pessimistic_lock |-> TRUE
+                    primary |-> l.primary,
+                    resolving_pessimistic_lock |-> l.type = "lock_key"
                     ]})
-        /\ UNCHANGED <<resp_msgs, client_vars, key_vars, next_ts>>
-        \/ /\ SendReqs({[type |-> "check_txn_status_req",
-                    start_ts |-> l.ts,
-                    primary |-> l.primary, 
-                    rollback_if_not_exist |-> FALSE,
-                    resolving_pessimistic_lock |-> TRUE
-                    ]})
-           /\ UNCHANGED <<resp_msgs, client_vars, key_vars, next_ts>>
-        
-      \* the `resolving_pessimistic_lock` field is set to *FALSE*
-      [] l.type \in {"prewrite_optimistic", "prewrite_pessimistic"} ->
-        /\ SendReqs({[type |-> "check_txn_status_req",
-                     start_ts |-> l.ts,
-                     primary |-> l.primary, 
-                     rollback_if_not_exist |-> TRUE,
-                     resolving_pessimistic_lock |-> FALSE
-                     ]})
-        /\ UNCHANGED <<resp_msgs, client_vars, key_vars, next_ts>>
-        \/ /\ SendReqs({[type |-> "check_txn_status_req",
-                    start_ts |-> l.ts,
-                    primary |-> l.primary, 
-                    rollback_if_not_exist |-> FALSE,
-                    resolving_pessimistic_lock |-> FALSE
-                    ]})
-           /\ UNCHANGED <<resp_msgs, client_vars, key_vars, next_ts>>
+      /\ UNCHANGED <<resp_msgs, client_vars, key_vars, next_ts>>
 
 \* Clean up stale locks by checking the status of the primary key.  Commit
 \* the secondary keys if primary key is committed; otherwise rollback the
 \* transaction by rolling-back the primary key, and then also rollback the
 \* secondarys.
 ServerCheckTxnStatus ==

```

Comment:
Update the comment and explain why `check_txn_status_resp` equals to sending `resolve` command directly, and how they are mapped.

Reply:
comment updated.